scripts/dev_cli.sh

```diff
@@ -255,7 +256,10 @@ cmd_tests() {
 	$DOCKER_RUNTIME run \
 	       --workdir "$CTR_CLH_ROOT_DIR" \
 	       --rm \
-	       --privileged \
+	       --device /dev/kvm \
```

Comment:
Should we remove the `dev:dev` container mount, provided that we also pass `/dev/vfio` ?

Reply:
Why do we need to pass `/dev/vfio`?

Reply:
for the vfio integration test to run.

Reply:
But we only use some virtio devices for this test, so from a host perspective I'm not sure why we need `/dev/vfio`. VFIO is needed in L1, but this does not require that our container has /dev/vfio available, does it?

Reply:
haha, you're right :-) we don't pass anything from the host through VFIO. Sorry for the noise.
We should try removing the `dev:dev` mount though.

Reply:
IIUC the only thing that is making the CI failing on this PR is related to virtio-fs (all virtio-fs integration tests failed), which would explain why test_vfio is failing too.

Reply:
I found out that `/dev:/dev` was covering for `/dev/net/tun`, which I explicitely passed with `--device /dev/net/tun`.

Reply:
I'm gonna drop this change for integration test, I cannot make the PR passing.